#! /bin/sh /usr/share/dpatch/dpatch-run
## 02-set-library-path-for-tests.dpatch by Tiago Saboga <tiagosaboga@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Set library path for junit tests.

@DPATCH@
diff -urNad libjna-java-3.0.4~/build.xml libjna-java-3.0.4/build.xml
--- libjna-java-3.0.4~/build.xml	2008-07-24 22:45:15.000000000 -0300
+++ libjna-java-3.0.4/build.xml	2008-07-24 22:49:56.000000000 -0300
@@ -402,6 +402,7 @@
     <property name="tests.platform" value=""/>
     <junit printsummary="yes" fork="${test.fork}" failureproperty="testfailure">
       <jvmarg value="-Djna.library.path=${build.native}"/>
+      <jvmarg value="-Djava.library.path=${build.native}:${java.library.path}"/>
       <!-- Avoid VM crashes, if possible -->
       <jvmarg value="-Djna.protected=true"/>
       <jvmarg value="-Ddir.build.suffix=${dir.build.suffix}"/>
